src/routes/likernft/book/store.ts

```diff
@@ -43,12 +43,14 @@ router.get('/list', jwtOptionalAuth('read:nftbook'), async (req, res, next) => {
   try {
     const {
       wallet,
+      excluded_wallet: excludedWallet,
```

Comment:
```suggestion
      exclude_wallet: excludedWallet,
```
Using verb seems better for qs

Reply:
But are `wallet`, `limit`, and `key` also a noun?

Reply:
it's like `ignore_whitespace=1` vs `ignored_whitespace=1`